test/NamespacePathMapperTest.php

```diff
@@ -40,6 +46,19 @@ public function testGetTestNamespaceForNamespace()
             'De\Idrinth\TestGenerator\Test\Abc',
             $object->getTestNamespaceForNamespace('De\Idrinth\TestGenerator\Test\Abc')
         );
+        $this->assertEquals(
+            'De\Test\Idrinth\AnyTestGenerator\Abc',
+            $object->getTestNamespaceForNamespace('De\Idrinth\AnyTestGenerator\Abc')
+        );
+    }
+
+    /**
+     * @test
+     * @expectedException \UnexpectedValueException
+     */
+    public function testGetTestNamespaceForNamespaceThrowsUnexpectedValueException()
+    {
+        $this->getInstance()->getTestNamespaceForNamespace('Der\Idrinth\hat\einen\Test-Generator');
```

Comment:
Der? Nenenene

Reply:
Oder ist das beabsichtigt für die UnexpectedValueException? Dann hab ich nix gesagt

Reply:
Ok, hätte den namespace mal komplett lesen sollen

Reply:
jupp, der Fall soll die exception produzieren, deswegen ein Namespace, den es nicht in der Liste gibt